docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -36,302 +49,360 @@ factors:
-rewards. Once properly deployed, you, as the owner, are not required to do anything else with your
-5G Hotspot to earn MOBILE.
+With the [passing](https://heliumvote.com/14jH67zhctwb3B5NmwiAjaXQuyF7jZMZCAnfyBYhSpRS3L22sQE) of
+[HIP-74](https://github.com/helium/HIP/blob/main/0074-mobile-poc-modeled-coverage-rewards.md)
+Modeled Coverage aims to allow for more appropriate Rewards and incentivizes the deployment of
+Radios at optimal locations.
 
-More than one Small Cell Radio can be connected to a single 5G Hotspot to create a larger 5G
-coverage area. 5G Hotspots can support as many Small Cell Radios as would saturate their bandwidth.
+This new algorithm uses the location of the Radio to calculate MOBILE Rewards based on the number of
+res12 hexes receiving coverage from a given Radio.
 
-Connecting more than one Small Cell Radio requires an additional networking switch. Refer to the 5G
-Hotspot Maker installation guides for specific instructions.
+The MOBILE Rewards Oracle will calculate a coverage model for each Radio by considering the physical
+location, antenna direction, radio max transmit power, and Obstruction Data Oracle inputs to
+calculate the number of res12 hexes receiving coverage from each Radio.
 
-:::info
+Depending on the number of res12 hexes and the resulting signal power in each, the MOBILE Rewards
+Oracle will allot **Coverage Points** to a given Radio for calculating its MOBILE Rewards instead of
+assuming a fixed Radio Type multiplier Reward.
+
+### Reward Hexagon Sizing
 
-During [Genesis](./mobile-poc#mobile-genesis), 5G Hotspots earn MOBILE by reporting their status via
-a "heartbeat," an automatic reporting process to the Helium Blockchain that the Small Cell Radio is
-online and registered with the SAS.
+Analysis of the coverage by a single Outdoor Radio with Modeled Coverage data shows that the res8
+hex used in the IOT Network is too large to represent a Radio's coverage. Using res12 hexes better
+aligns with what an Outdoor Radio can reasonably cover, with an average area of 0.0003071 km² and an
+edge length of 9.4 m, slightly bigger than an average single-family home.
 
-Genesis is the period during which Hotspot owners are rewarded for providing 5G coverage with
-pre-mined MOBILE tokens. MOBILE tokens will be redeemable for HNT after the Genesis period ends.
+To summarize:
+
+- Hexes in the MOBILE Network identify the level of coverage from all the different Radios providing
+  coverage to each res12 hex.
+- A Radio can provide coverage for a hex it is not located within.
+- A Radio can earn Rewards by providing coverage in multiple hexes.
+
+To help visualize the significant difference between these two resolutions, below is a map with a
+green res8 hex and a purple res12 hex inside.
+
+<img
+  className="explorerSidebar"
+  src={useBaseUrl('img/5g-on-helium/hex-12-and-hex-8-comparison.png')}
+/>
+
+### Modeled Coverage Reward Tiers
+
+:::info Heartbeat impact on Radio rewards
```

Comment:
Heartbeat impact title seemed to be unrelated to the explainer text.

Reply:
agreed, removed!